repo.go

```diff
@@ -625,10 +625,12 @@ func (r *Repository) downloadAndSaveItem(client Client, downloadingItem *downloa
 	downloadingItem.path = r.fullPath(it.filePath)
 	downloadingItem.pathMu.Unlock()
 
-	// try a few times in case of network trouble
 	var h hash.Hash
 	var x *exif.Exif
-	for i := 0; i < 3; i++ {
+
+	attempt := 1
+
+	for {
```

Comment:
It's all about Go's `:=` operator. It's too unobvious. `outFile, err := os.Create(downloadingItem.path)` creates a new `err` variable and `if err != nil` outside the `for` loop doesn't work as expected. I could rewrite it without changing `for i := 0; i < 3; i++ { ... }` to `for { ... }`, but this version looks more simple and explicit to me.